ceph-installer-rpm/build/build

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+set -ex
+
+sudo yum -y install epel-release
+sudo yum -y install fedpkg
+
+# Mock configuration file.
+# TODO: Use the CentOS Storage SIG repository instead of this Fedora Copr repo.
+# (discussion ongoing on the centos-devel list)
+MOCK_CFG=https://fedorapeople.org/~ktdreyer/ceph-installer/ktdreyer-ceph-installer.cfg
+
+# Install this mock configuration file.
+if [ ! -f /etc/mock/$(basename $MOCK_CFG) ]; then
+  # We only refresh the config file in /etc/mock if it does not already exist.
+  # If we were to refresh this file on every run, we would bump the timestamp
+  # each time, and mock would see the timestamp change and invalidate its
+  # cache. By checking the file in /etc/mock first, we can save a bit of build
+  # time here.
+  curl $MOCK_CFG | sudo tee /etc/mock/$(basename $MOCK_CFG)
+fi
+
+# Add the Jenkins slave UID to the mock group.
+sudo usermod -a -G mock $(whoami)
+
+# Attempt the build. If it fails, print the mock logs to STDOUT.
+make rpm || ( tail -n +1 {root,build}.log && exit 1 )
+
+BRANCH=`branch_slash_filter $BRANCH`
+
+# Upload to chacra:
+virtualenv venv
+venv/bin/pip install chacractl
+echo venv/bin/chacractl binary create ceph-installer/$BRANCH/centos/7/noarch /var/lib/mock/epel-7-x86_64/result/*.noarch.rpm
```

Comment:
Is this `echo` left on purpose? 

I don't think `chacractl` can consume globs like this also. We usually just call `find` and then pipe into chacractl as stdin